link/kprobe.go

```diff
@@ -157,7 +157,7 @@ func kprobe(symbol string, prog *ebpf.Program, ret bool) (*perfEvent, error) {
 // pmuKprobe opens a perf event based on the kprobe PMU.
 // Returns os.ErrNotExist if the given symbol does not exist in the kernel.
 func pmuKprobe(symbol string, ret bool) (*perfEvent, error) {
-	return pmuProbe(kprobeType, symbol, "", 0, ret)
+	return pmuProbe(kprobeType, symbol, "", 0, ret, perfAllThreads)
```

Comment:
This is my OCD talking, but could we keep `ret` as the final parameter across the board out of convention?

Reply:
sure, I'll change it